python_modules/dagster/dagster/_core/definitions/asset_condition/asset_condition_evaluation_context.py

```diff
@@ -220,7 +226,10 @@ def materialized_since_previous_tick_subset(self) -> ValidAssetSubset:
     def previous_tick_requested_subset(self) -> AssetSubset:
         """The set of asset partitions that were requested (or discarded) on the previous tick."""
         if (
-            self.previous_evaluation_state is None
+            # if we're not in the context of a scheduling policy evaluation, it is unknown which
+            # partitions were requested on the previous tick
+            not self.is_scheduling_policy_evaluation
+            or self.previous_evaluation_state is None
```

Comment:
This seems not great. It is possible instead to simulate the state that would happen in production?

Reply:
reworked this bit -- I think the ideal end state might be one where we don't do this inference that "top level condition was true on previous tick = that asset partition was requested", and rather just do explicit checks for partitions being in progress / failed etc. but that has some larger implications. For now, reworked such that we just make sure the top level condition is always false so we don't have to think about this.